config_auth.go

```diff
@@ -2,13 +2,26 @@ package evergreen
 
 import (
 	"github.com/evergreen-ci/utility"
+	"github.com/mongodb/anser/bsonutil"
 	"github.com/mongodb/grip"
 	"github.com/pkg/errors"
 	"go.mongodb.org/mongo-driver/bson"
 	"go.mongodb.org/mongo-driver/mongo"
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
+var (
+	AuthLDAPKey                    = bsonutil.MustHaveTag(AuthConfig{}, "LDAP")
+	AuthOktaKey                    = bsonutil.MustHaveTag(AuthConfig{}, "Okta")
+	AuthGithubKey                  = bsonutil.MustHaveTag(AuthConfig{}, "Github")
+	AuthNaiveKey                   = bsonutil.MustHaveTag(AuthConfig{}, "Naive")
+	AuthOnlyAPIKey                 = bsonutil.MustHaveTag(AuthConfig{}, "OnlyAPI")
+	AuthMultiKey                   = bsonutil.MustHaveTag(AuthConfig{}, "Multi")
+	authPreferredTypeKey           = bsonutil.MustHaveTag(AuthConfig{}, "PreferredType")
```

Comment:
Why are L20 and L21 unexported?

Reply:
They don't have to be exported. The others are necessary for the auth package.